migrations/versions/schema/2024-06-11_1b485cfea981_add_roles.py

```diff
@@ -34,7 +34,7 @@ def upgrade() -> None:
     conn.execute(
         sa.text(
             """
-            INSERT INTO shops (id, name, description, vat_standard) VALUES (:id, 'shop', 'Default shop', 21.0)
+            INSERT INTO shops (id, name, description, vat_standard, vat_lower_1, vat_lower_2, vat_lower_3, vat_special, vat_zero) VALUES (:id, 'shop', 'Default shop', 21.0, 15.0, 10.0, 5.0, 2.0, 0.0)
```

Comment:
Rewriting an old migration will only fix test; it won't be executed on prod -> as this migrations has already been executed,

Reply:
Please fix

Reply:
DB on prod was lacking these values.
New shops seem to have it.